MODULE.bazel

```diff
@@ -25,13 +25,14 @@ bazel_dep(name = "supply_chain_tools", version = "0.0.1")
 bazel_dep(name = "platforms", version = "1.0.0")
 bazel_dep(name = "bazel_features", version = "1.34.0")
 bazel_dep(name = "bazel_skylib", version = "1.8.2")
+bazel_dep(name = "bazel_lib", version = "3.0.0")
 bazel_dep(name = "rules_pkg", version = "1.1.0")
 bazel_dep(name = "rules_shell", version = "0.6.1")
 
 # Temporary until a future rules_pkg release
 git_override(
     module_name = "rules_pkg",
-    commit = "4eed5466fb7062e196c4fa01495e1dfe0cf5c850",
+    commit = "6cdaba69ee76463b2b8e97e8d243dbb6115c3aee",
```

Comment:
Latest is now:
daa496412d12598d6b24619a9a6cd6fc137de384

That has some recent changes from Regis.

Reply:
Done.